newapi/ooniapi/auth.py

```diff
@@ -411,23 +428,35 @@ def set_account_role():
 
 
 def _delete_account_data(email_address: str) -> None:
+    # Used by integ test
     account_id = hash_email_address(email_address)
-    query = "DELETE FROM accounts WHERE account_id = :account_id"
     query_params = dict(account_id=account_id)
-    q = current_app.db_session.execute(query, query_params).rowcount
-    current_app.db_session.commit()
-    # TODO return status
-    return q
+    if current_app.config["DATABASE_URI_RO"]:
+        query = "DELETE FROM accounts WHERE account_id = :account_id"
+        q = current_app.db_session.execute(query, query_params).rowcount
+        current_app.db_session.commit()
+
+    if current_app.config["USE_CLICKHOUSE"]:
+        # reset account to "user" role
+        q = "INSERT INTO accounts (account_id, role) VALUES(:account_id, 'role')"
```

Comment:
Why is the value of `role` set to `'role'`?

Also, this isn't really equivalent to deleting the row from the DB. Perhaps we could make use of the DELETE mutation?
https://clickhouse.com/docs/en/sql-reference/statements/alter/delete/

Reply:
Is there the possibility of deleting an item in this key value store?

Reply:
But why is the value of `role` set to `'role'` instead of `'user'`?

Reply:
Added a FIXME